internal/handlers/telegram/hander_test.go

```diff
@@ -0,0 +1,57 @@
+package telegram
+
+import (
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api"
+	"github.com/kyokomi/emoji"
+	"github.com/ritlug/teleirc/internal"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestStickerSmile(t *testing.T) {
```

Comment:
So this is a good start, but this only tests if everything is working correctly. What if we have a sticker object, but there isn't an emoji present? What if the `UserName` isn't present?

Reply:
How would there be a sticker with no emoji?